providers/aws/services/cw/resource.ftl

```diff
@@ -44,6 +44,15 @@
     /]
 [/#macro]
 
+[#macro createVPCLogGroup id name retention ]
```

Comment:
Do we need this one? looks like it just creates a Log Group

Reply:
Removed.